tools/install_tensorflow_cc.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+
+export JAVA_HOME=/export/b02/hxu/TensorFlow/java/jdk1.8.0_121
```

Comment:
I don't think this export statement belongs here, at least not in the checked-in version.